src/xdocs/beginning_development.xml

```diff
@@ -31,6 +31,14 @@
             git clone git@github.com:you_user_name/checkstyle.git
         </source>
       </div>
+      <p>
+        4. Before opening project in IDEA do build in terminal by running in repository root folder
+      </p>
+      <div class="wrap-content">
+        <source>
+            mvn install
```

Comment:
I am more curious why `mvn install` is needed. I never installed checkstyle snapshot outside of regression and never had any issues. Should this be something more like `mvn verify test`?

Reply:
It runs all phases of maven.
If this command pass, everything that contributor might run is ok and doable at command line.

Reply:
>mvn verify test

This is not correct command, it runs tests twice.

I just tried to be very simple in my command.
`install` is required for many no-error CI executions

Reply:
Ok.